parser/src/test/java/org/dst/parse/Test.java

```diff
@@ -0,0 +1,46 @@
+package org.dst.parse;
```

Comment:
1. Please move the test files to `test` module.
2. Use testng instead of testng since our codebase is using testng now.